File: mobile/busapp/lib/services/models/hotel_model.dart
class HotelModel {
  final int id;
  final String title;
  final String address;
  final String phone;
  final String email;
  final int stars;

  HotelModel({
    this.id,
    this.title,
    this.address,
    this.phone,
    this.email,
    this.stars,
  });

  HotelModel.fromJson(Map<String, dynamic> json)
      : id = json['id'],
        title = json['title'],
        address = json['address'],
        phone = json['phone'],
        email = json['email'],
        stars = json['stars'];

  Map<String, dynamic> toJson() => {
        'id': id,
        'title': title,
        'address': address,
        'phone': phone,
        'email': email,
        'stars': stars,
      };

  @override
  String toString() {
    return '''id: $id, title: $title, address: $address, phone: $phone, email: $email, stars: $stars''';
  }
}

File: microservices/hotels/src/main/java/kz/placer/hotels/hotels/models/HotelModel.java
package kz.placer.hotels.hotels.models;

import com.fasterxml.jackson.annotation.JsonIgnore;
import com.sun.istack.NotNull;
import lombok.Data;

import javax.persistence.*;
import java.util.ArrayList;
import java.util.List;

@Entity
@Table(name = "hotels")
@Data
public class HotelModel{

	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	private Integer id;
	@NotNull
	private String title;
	@NotNull
	private String address;
	@NotNull
	private String phone;
	@NotNull
	private String email;
	@NotNull
	private int stars;
	private String avatar;
}
